main.go

```diff
@@ -134,8 +134,8 @@ func main() {
 		inventory.Run()
 		tasker.Close()
 		return
-	case "ospackage":
-		ospackage.Run(ctx, config.Instance())
+	case "policies":
```

Comment:
since this will be in our docs, can we support "guestpolicies" too? And consider continuing to support "ospackage" for now so existing users don't need to re-enable the feature.